Fixed values updation on Capacity on Demand page

tiwari-nishant opened the pull request:
•  Fixed refetching of values for capacity on demand page  on new license activation.
•  Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=638365

tiwari-nishant committed:
• Fixed refetching of values for capacity on demand page  on new license activation.
• Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=638365

tiwari-nishant committed:
• Fixed refetching of values for capacity on demand page  on new license activation.
• Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=638365

Nikhil-Ashoka commented:
Keep the heading short

tiwari-nishant commented:
done @Nikhil-Ashoka

Nikhil-Ashoka reviewed (changes requested):
We can optimize the code here by calling **fetchInfo()** in the created method also.

tiwari-nishant committed:
Merge branch '1110' of github.com:ibm-openbmc/webui-vue into 1110

tiwari-nishant committed:
Merge branch 'COD_Fix' of https://github.com/tiwari-nishant/webui-vue into COD_Fix

tiwari-nishant committed:
• Fixed refetching of values for capacity on demand page on new license activation.

• Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=638365

Signed-off-by: tiwari-nishant <tiwari.nishant@ibm.com>

tiwari-nishant committed:
• Fixed refetching of values for capacity on demand page  on new license activation.
• Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=638365
